src/Plugin/Field/FieldWidget/InlineEntityFormMultiple.php

```diff
@@ -240,7 +240,6 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $element['#element_validate'] = array('inline_entity_form_update_row_weights');
     // Add the required element marker & validation.
     if ($element['#required']) {
-      $element['#title'] .= ' ' . _theme('form_required_marker', array('element' => $element));
```

Comment:
I believe this doesn't apply to title field. It applies to IEF as a whole. I don't see * there if I make entity reference field required.